features/convergedcloud/exec.config

```diff
@@ -3,12 +3,10 @@ set -Eeuo pipefail
 
 systemctl enable ignition-disable.service
 
-# crictl
-CRI_VERSION=1.26.1
+K8S_VERSION=v1.30.7
+K8S_VERSION_REPO="${K8S_VERSION%.*}"
 
-wget --show-progress --progress=bar:force -4 -P /usr/local/bin \
-  https://github.com/kubernetes-sigs/cri-tools/releases/download/v${CRI_VERSION}/crictl-v${CRI_VERSION}-linux-amd64.tar.gz
-
-rm -f /root/.wget-hsts
-
-tar -C /usr/local/bin -xf /usr/local/bin/crictl-v${CRI_VERSION}-linux-amd64.tar.gz
+gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg </builder/features/khost/release.key
```

Comment:
As you do not use the same version as the feature `khost` uses this feature should ship it's own `release.key` compatible and trusted for `v1.30.7`.

Reply:
I agree, and I guess we could even reuse the already available apt source

Reply:
renamed the file and added the release key

Reply:
khost feature was only introduced for the key hence this was removed again (same for chost which is a dependency for khost). The change from the gardener feature to chost will be done in a separate PR. khost feature will not be used at all.